src/utils/src/Allocators.c

```diff
@@ -53,7 +53,7 @@ VOID dumpMemoryHex(PVOID pMem, UINT32 size)
     PCHAR pCur = buf;
     PBYTE pByte = (PBYTE) pMem;
     for (UINT32 i = 0; i < size; i++) {
-        SPRINTF(pCur, "%02x ", *pByte++);
+        SNPRINTF(pCur, 2, "%02x ", *pByte++);
```

Comment:
Isnt this supposed to be 3 to include the NULL character?
https://cplusplus.com/reference/cstdio/snprintf/